### PR TITLE
Chinese translation of Build

### PR DESCRIPTION
**Pull request by cool007zqw**

### CODE REVIEW

**Comment by lsaether** [comment]
Thanks! We will review this shortly.

File: docs/lang/chinese/build/Hackathons.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## 黑客马拉松
+
+由于我们Web3基金会的目标之一是要推动[Web3技术栈](http://wiki.web3.foundation/en/latest/tech_stack/tech_stack_overview/)，我们支持任何想要在各自城市组织黑客马拉松，从而让更多开发者能够理解Polkadot/Substrate的个人或者团体。
```
Comment (ansonla3) [inline docs/lang/chinese/build/Hackathons.md]:
```suggestion
由于我们 Web3 基金会的目标之一是要推动[Web3技术栈](http://wiki.web3.foundation/en/latest/tech_stack/tech_stack_overview/)，我们支持任何想要在各自城市组织黑客马拉松，从而让更多开发者能够理解 Polkadot / Substrate 的个人或者团体。
```

File: docs/lang/chinese/build/Hackathons.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+我们不仅会为您提供技术资源，而且我们团队还会派人组织关于该技术的实践研讨会/演讲，帮助参与者们更快地精通于此。
+
+## 想法
+
+- 非同质化代币
+- 抵押代币
+- 有着不同投票和民主风格的新管理模式
+- 更新TCR模块
```
Comment (ansonla3) [inline docs/lang/chinese/build/Hackathons.md]:
```suggestion
- 更新 TCR 模块
```

File: docs/lang/chinese/build/Hackathons.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- IPFS集成
```
Comment (ansonla3) [inline docs/lang/chinese/build/Hackathons.md]:
```suggestion
- IPFS 集成
```

File: docs/lang/chinese/build/Hackathons.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- 去中心化身份解决方案
+- 预言家模块
+- 实施普遍的状态通道/plasma Chain
```
Comment (ansonla3) [inline docs/lang/chinese/build/Hackathons.md:16]:
```suggestion
- 实施通用状态通道 / 离子网络链
```

File: docs/lang/chinese/build/Hackathons.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- 去中心化交易所
+- 用于监管验证者表现的工具
+- 跨越不同平行链的Dapp
+
+## 为什么要举办黑客马拉松？
+
+- 热衷于推广Web3版本
```
Comment (ansonla3) [inline docs/lang/chinese/build/Hackathons.md:23]:
```suggestion
- 热衷于推广Web3愿景
```

File: docs/lang/chinese/build/build-with-polkadot.md.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+为此，Parity Technologies正积极开发Substrate的扩展-Cumulus，它将使任何基于Substrate建立的链易于转移到Polkadot兼容的平行链上。
+
+#### Cumulus
+
+[Cumulus](./cumulus.md)旨在成为Substrate的扩展，它将使所有Substrate运行时兼容于Polkadot。
+
+它将处理网络兼容性，供平行链用于实现以连接到Polkadot网络上，其中包括：
+
+- 跨链信息传递
+- 只需稍微修改（可能只需导入crate和添加一行代码
```
Comment (ansonla3) [inline docs/lang/chinese/build/build-with-polkadot.md.md:98]:
```suggestion
- 只需稍微修改 (可能只需导入crate和添加一行代码)
```

File: docs/lang/chinese/build/examples/index.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+### Parity
+
+- [Shasper](https://github.com/paritytech/shasper)——作为Substrate运行时运行Eth2.0
+- [Substrate TCR](https://github.com/parity-samples/substrate-tcr)——将TCR作为运行时模组
+- [Substrate ERC20](https://github.com/parity-samples/substrate-erc20)——在Substrate上执行ERC20代币标准
+
+### 社区
+
+- 使用Substrate的AdEx支付渠道。[视频](https://www.youtube.com/watch?v=1CeI6Oa1BnU)，[源码仓库](https://github.com/Polygos/substrate-node-cidchain)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:15]:
```suggestion
- 使用Substrate的AdEx支付渠道。[视频](https://www.youtube.com/watch?v=1CeI6Oa1BnU)，[源码库](https://github.com/Polygos/substrate-node-cidchain)
```

File: docs/lang/chinese/build/examples/index.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+- 用于与ROS集成的Robonomics模式。[源码仓库](https://github.com/airalab/substrate-node-robonomics)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:16]:
```suggestion
- 用于与ROS集成的Robonomics模式。[源码库](https://github.com/airalab/substrate-node-robonomics)
```

File: docs/lang/chinese/build/examples/index.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+- Staked Technologies在Substrate上实现Plasma。[源码仓库](https://github.com/stakedtechnologies/Plasm)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:17]:
```suggestion
-  Staked Technologies在Substrate上实现Plasma。[源码库](https://github.com/stakedtechnologies/Plasm)
```

File: docs/lang/chinese/build/examples/index.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+- Edgeware。[源码仓库](https://github.com/hicommonwealth/edgeware-node)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:18]:
```suggestion
- Edgeware。[源码仓库](https://github.com/hicommonwealth/edgeware-node)
```

File: docs/lang/chinese/build/examples/index.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+- 专注于隐私安全的运行时模组的Zerochain
+	- [Zerochain 源码仓库]()
+	- [Bellman-substrate（substrate上的zksnarks）源码仓库](https://github.com/LayerXcom/bellman-substrate)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:21]:
```suggestion
- 专注于隐私安全的运行时模组的Zerochain
	- [Zerochain 源码库](https://github.com/LayerXcom/zero-chain)
	- [Bellman-substrate（substrate上的zksnarks）源码库](https://github.com/LayerXcom/bellman-substrate)
```

File: docs/lang/chinese/build/examples/index.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+- Substrate上的Starlog，IPFS元数据。[源码仓库](https://github.com/PACTCare/Starlog)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:22]:
```suggestion
- Substrate上的Starlog，IPFS元数据。[源码库](https://github.com/PACTCare/Starlog)
```

File: docs/lang/chinese/build/examples/index.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+- 案例：苹果应用商店使用substrate出售苹果产品。[源码仓库](https://github.com/osuketh/apple-store-substrate)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:23]:
```suggestion
- 案例：苹果应用商店使用substrate出售苹果产品。[源码库](https://github.com/osuketh/apple-store-substrate)
```

File: docs/lang/chinese/build/examples/index.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+- Darwinia，一个跨链游戏平台。[源码仓库](https://github.com/darwinia-network/darwinia)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:24]:
```suggestion
- Darwinia，一个跨链游戏平台。[源码库](https://github.com/darwinia-network/darwinia)
```

File: docs/lang/chinese/build/examples/index.md [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+- Joystream，一个用户管理视频平台。 [源码仓库](https://github.com/Joystream/substrate-node-joystream)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:25]:
```suggestion
- Joystream，一个用户管理视频平台。 [源码库](https://github.com/Joystream/substrate-node-joystream)
```

File: docs/lang/chinese/build/examples/index.md [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+- IPFS浏览器。[源码仓库](https://github.com/Polygos/substrate-node-ipfsbrowser)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:26]:
```suggestion
- IPFS浏览器。[源码库](https://github.com/Polygos/substrate-node-ipfsbrowser)
```

File: docs/lang/chinese/build/examples/index.md [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+- Substrate多重签名。源码仓库](https://github.com/mixbytes/substrate-module-multisig)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:27]:
```suggestion
- Substrate多重签名。[源码库](https://github.com/mixbytes/substrate-module-multisig)
```

File: docs/lang/chinese/build/examples/index.md [hunk 18]
```diff
@@ -0,0 +1,33 @@
...
+- 默克尔树模块。[源码仓库](https://github.com/filiplazovic/substrate-merkle-tree)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:28]:
```suggestion
- 默克尔树模块。[源码库](https://github.com/filiplazovic/substrate-merkle-tree)
```

File: docs/lang/chinese/build/examples/index.md [hunk 19]
```diff
@@ -0,0 +1,33 @@
...
+- 使用substrate的停车位注册表。[源码仓库](https://github.com/yjkimjunior/ParkingSpaceSubstrate)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:29]:
```suggestion
- 使用substrate的停车位注册表。[源码库](https://github.com/yjkimjunior/ParkingSpaceSubstrate)
```

File: docs/lang/chinese/build/examples/index.md [hunk 20]
```diff
@@ -0,0 +1,33 @@
...
+- Substrate固有样本。[源码仓库](https://github.com/gautamdhameja/substrate-inherents-sample)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:30]:
```suggestion
- Substrate固有样本。[源码库](https://github.com/gautamdhameja/substrate-inherents-sample)
```

File: docs/lang/chinese/build/examples/index.md [hunk 21]
```diff
@@ -0,0 +1,33 @@
...
+- 抵押非同质化资产以获取现金。[源码仓库](https://github.com/nczhu/collateral)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:31]:
```suggestion
- 抵押非同质化资产以获取现金。[源码库](https://github.com/nczhu/collateral)
```

File: docs/lang/chinese/build/examples/index.md [hunk 22]
```diff
@@ -0,0 +1,33 @@
...
+- CidChain。[源码仓库](https://github.com/Polygos/substrate-node-cidchain)
```
Comment (ansonla3) [inline docs/lang/chinese/build/examples/index.md:32]:
```suggestion
- CidChain。[源码库](https://github.com/Polygos/substrate-node-cidchain)
```

File: docs/lang/chinese/build/index.md [hunk 23]
```diff
@@ -0,0 +1,45 @@
...
+
+在这里，您可以了解@(Polkadot)生态系统中开发工具的最新状态。我们一直致力于学习新的工具和框架，并将它们添加进网站内容中。如您有认为应包含在内的信息，请在[Riot](https://riot.im/app/#/room/#polkadot-watercooler:matrix.org)上联系我们。
+
+我们将本维基网站的“构建者”章节分为三部分：第一部分为技术指南，帮助您快速了解Polkadot的开发进度。第二部分为最新的开发者工具目录。第三部分中的资源将包含更深层次的、其它地方找不到的技术信息。
+
+## 主页
+
+## 开发指南
+
+- [入门指南](./build-with-polkadot.md)——对平行链和智能合约的选择，以及目前可用资源进行高度概述。
```
Comment (ansonla3) [inline docs/lang/chinese/build/index.md:16]:
```suggestion
- [入门指南](./build-with-polkadot.md) - 对平行链和智能合约的选择，以及目前可用资源进行高度概述。
```

File: docs/lang/chinese/build/index.md [hunk 24]
```diff
@@ -0,0 +1,45 @@
...
+
+- [开发路线图](./dev-roadmap.md)
+
+- [平行链开发工具包](./pdk.md)——了解平行链开发工具包并创建平行链
```
Comment (ansonla3) [inline docs/lang/chinese/build/index.md:20]:
```suggestion
- [平行链开发工具包](./pdk.md) - 了解平行链开发工具包并创建平行链
```

File: docs/lang/chinese/build/index.md [hunk 25]
```diff
@@ -0,0 +1,45 @@
...
+
+- [智能合约](./smart-contracts.md)——学习Polkadot和Substrate中的所有智能合约，以及现在开始构建的指南
```
Comment (ansonla3) [inline docs/lang/chinese/build/index.md:22]:
```suggestion
- [智能合约](./smart-contracts.md) - 学习Polkadot和Substrate中的所有智能合约，以及现在开始构建的指南
```

File: docs/lang/chinese/build/index.md [hunk 26]
```diff
@@ -0,0 +1,45 @@
...
+
+## 工具
+
+- [工具](./tools/index.md)——受保护的工具清单。
```
Comment (ansonla3) [inline docs/lang/chinese/build/index.md:26]:
```suggestion
- [工具](./tools/index.md) - 受保护的工具清单。
```

File: docs/lang/chinese/build/index.md [hunk 27]
```diff
@@ -0,0 +1,45 @@
...
+
+- [子密钥](./tools/subkey.md)——用于生成Polkadot/Substrate密钥的命令行实用程序。
```
Comment (ansonla3) [inline docs/lang/chinese/build/index.md:28]:
```suggestion
- [子密钥](./tools/subkey.md) - 用于生成Polkadot/Substrate密钥的命令行实用程序。
```

File: docs/lang/chinese/build/index.md [hunk 28]
```diff
@@ -0,0 +1,45 @@
...
+
+## 资源
+
```
Comment (ansonla3) [inline docs/lang/chinese/build/index.md:31]:
```suggestion
交易所整合(./exchange-integration.md)
```

File: docs/lang/chinese/build/networks.md [hunk 29]
```diff
@@ -0,0 +1,67 @@
...
+
+通过在终端上运行，您可以在机器上跑简单的单节点开发网络：
+
+```bash
+polkadot --dev
+```
+
+您可以复制和创建以下网站：
+
+http://github.com/paritytech/polka-ui and http://github.com/paritytech/polkadot-ui or just heading to https://polkadot.js.org/apps.
```
Comment (ansonla3) [inline docs/lang/chinese/build/networks.md:21]:
```suggestion
http://github.com/paritytech/polka-ui and http://github.com/paritytech/polkadot-ui 或前往到 https://polkadot.js.org/apps.
```

File: docs/lang/chinese/build/networks.md [hunk 30]
```diff
@@ -0,0 +1,67 @@
...
+
+Alexander是一个在PoC-3和PoC-4开发期间使用的测试网络。
+
+请在以下`路径`安装Polkadot Poc-4和`polkadot`二进制文件：
+
+```bash
+cargo install --git https://github.com/paritytech/polkadot.git --branch v0.4 polkadot
+```
+
+通过默认运行以下代码来连接到全球“Alexander测试网：”
```
Comment (ansonla3) [inline docs/lang/chinese/build/networks.md:47]:
```suggestion
通过默认运行以下代码来连接到全球"Alexander测试网:"
```

File: docs/lang/chinese/build/smart-contracts.md [hunk 31]
```diff
@@ -0,0 +1,44 @@
+# 智能合约
+
+Polkadot中继链本身并不支持智能合约，不过在它之上的平行链可以。目前已出现[Edgeware](https://edgewa.re)等多个项目，而Substrate的内置[合约模块]((https://crates.parity.io/srml_contract/index.html))将很可能使得更多的平行链支持这一特性。
+
+## 资源
+
+下文列出了开发者当前可使用的资源，帮助开始编写智能合约并部署在基于Substrate的平行链之上。
+
+- [ink!](https://github.com/paritytech/ink)——Parity的ink用于编写智能合约。
```
Comment (ansonla3) [inline docs/lang/chinese/build/smart-contracts.md:9]:
```suggestion
- [ink!](https://github.com/paritytech/ink) - Parity的ink用于编写智能合约。
```

File: docs/lang/chinese/build/smart-contracts.md [hunk 32]
```diff
@@ -0,0 +1,44 @@
+# 智能合约
+
+Polkadot中继链本身并不支持智能合约，不过在它之上的平行链可以。目前已出现[Edgeware](https://edgewa.re)等多个项目，而Substrate的内置[合约模块]((https://crates.parity.io/srml_contract/index.html))将很可能使得更多的平行链支持这一特性。
+
+## 资源
+
+下文列出了开发者当前可使用的资源，帮助开始编写智能合约并部署在基于Substrate的平行链之上。
+
+- [ink!](https://github.com/paritytech/ink)——Parity的ink用于编写智能合约。
+- [部署首个合约](https://github.com/paritytech/ink/wiki/Deploying-Your-First-Contract)——部署给定的`flipper`合约的指南。
```
Comment (ansonla3) [inline docs/lang/chinese/build/smart-contracts.md:10]:
```suggestion
- [部署首个合约](https://github.com/paritytech/ink/wiki/Deploying-Your-First-Contract) - 部署给定的`flipper`合约的指南。
```

File: docs/lang/chinese/build/smart-contracts.md [hunk 33]
```diff
@@ -0,0 +1,44 @@
...
+
+Polkadot中继链本身并不支持智能合约，不过在它之上的平行链可以。目前已出现[Edgeware](https://edgewa.re)等多个项目，而Substrate的内置[合约模块]((https://crates.parity.io/srml_contract/index.html))将很可能使得更多的平行链支持这一特性。
+
+## 资源
+
+下文列出了开发者当前可使用的资源，帮助开始编写智能合约并部署在基于Substrate的平行链之上。
+
+- [ink!](https://github.com/paritytech/ink)——Parity的ink用于编写智能合约。
+- [部署首个合约](https://github.com/paritytech/ink/wiki/Deploying-Your-First-Contract)——部署给定的`flipper`合约的指南。
+- [编写首份合同](https://github.com/paritytech/ink/wiki/Writing-Your-First-Contract)——编写`flipper`合约的指南。
```
Comment (ansonla3) [inline docs/lang/chinese/build/smart-contracts.md:11]:
```suggestion
- [编写首份合同](https://github.com/paritytech/ink/wiki/Writing-Your-First-Contract) - 编写`flipper`合约的指南。
```

File: docs/lang/chinese/build/smart-contracts.md [hunk 34]
```diff
@@ -0,0 +1,44 @@
...
+- [Substrate合约工作坊](https://shawntabrizi.github.io/substrate-contracts-workshop/#/)——帮助了解使用`ink!`和编写ERC20代币的基本方法。
```
Comment (ansonla3) [inline docs/lang/chinese/build/smart-contracts.md:12]:
```suggestion
- [Substrate合约工作坊](https://shawntabrizi.github.io/substrate-contracts-workshop/#/) - 帮助了解使用`ink!`和编写ERC20代币的基本方法。
```

File: docs/lang/chinese/build/tools/index.md [hunk 35]
```diff
@@ -0,0 +1,82 @@
+# 工具
+
+在这里，我们提供了满足您开发需求的工具列表，它将按照编程语言进行分类。目前，Rust生态系统最为成熟，您可以现在就开始在Substrate上进行构建。随着围绕其它编程语言的生态系统不断扩展，我们将陆续完善以下章节。
+
+## 区块资源管理器
+
+- [Polka.io](https://polka.io)——用于Polkadot的区块链资源管理器。
```
Comment (ansonla3) [inline docs/lang/chinese/build/tools/index.md:7]:
```suggestion
- [Polka.io](https://polka.io) - 用于Polkadot的区块链资源管理器。
```

File: docs/lang/chinese/build/tools/index.md [hunk 36]
```diff
@@ -0,0 +1,82 @@
+# 工具
+
+在这里，我们提供了满足您开发需求的工具列表，它将按照编程语言进行分类。目前，Rust生态系统最为成熟，您可以现在就开始在Substrate上进行构建。随着围绕其它编程语言的生态系统不断扩展，我们将陆续完善以下章节。
+
+## 区块资源管理器
+
+- [Polka.io](https://polka.io)——用于Polkadot的区块链资源管理器。
+- [Polkadot-JS Apps Explorer](https://polkadot.js.org/apps/#/explorer)——Polkadot仪表板区块资源管理器。当前默认连接到Alexander测试网，您可以自行配置以连接到其它远程或本地端点。
```
Comment (ansonla3) [inline docs/lang/chinese/build/tools/index.md:8]:
```suggestion
- [Polkadot-JS Apps Explorer](https://polkadot.js.org/apps/#/explorer) - Polkadot仪表板区块资源管理器。当前默认连接到Alexander测试网，您可以自行配置以连接到其它远程或本地端点。
```

File: docs/lang/chinese/build/tools/index.md [hunk 37]
```diff
@@ -0,0 +1,82 @@
+# 工具
+
+在这里，我们提供了满足您开发需求的工具列表，它将按照编程语言进行分类。目前，Rust生态系统最为成熟，您可以现在就开始在Substrate上进行构建。随着围绕其它编程语言的生态系统不断扩展，我们将陆续完善以下章节。
+
+## 区块资源管理器
+
+- [Polka.io](https://polka.io)——用于Polkadot的区块链资源管理器。
+- [Polkadot-JS Apps Explorer](https://polkadot.js.org/apps/#/explorer)——Polkadot仪表板区块资源管理器。当前默认连接到Alexander测试网，您可以自行配置以连接到其它远程或本地端点。
+- Polkascan——用于Polkadot的区块链资源管理器。**目前下线**
```
Comment (ansonla3) [inline docs/lang/chinese/build/tools/index.md:9]:
```suggestion
- Polkascan - 用于Polkadot的区块链资源管理器。**目前下线**
```

File: docs/lang/chinese/build/tools/index.md [hunk 38]
```diff
@@ -0,0 +1,82 @@
...
+## 区块资源管理器
+
+- [Polka.io](https://polka.io)——用于Polkadot的区块链资源管理器。
+- [Polkadot-JS Apps Explorer](https://polkadot.js.org/apps/#/explorer)——Polkadot仪表板区块资源管理器。当前默认连接到Alexander测试网，您可以自行配置以连接到其它远程或本地端点。
+- Polkascan——用于Polkadot的区块链资源管理器。**目前下线**
+
+## 钱包
+
+- [Polkadot-JS Apps——账户](https://polkadot.js.org/apps/#/accounts)
+- [Bonds oo7 Polkadot UI](https://github.com/paritytech/substrate-ui)——使用Bonds oo7库构建。
```
Comment (ansonla3) [inline docs/lang/chinese/build/tools/index.md:14]:
```suggestion
- [Bonds oo7 Polkadot UI](https://github.com/paritytech/substrate-ui) - 使用Bonds oo7库构建。
```

File: docs/lang/chinese/build/tools/index.md [hunk 39]
```diff
@@ -0,0 +1,82 @@
...
+- [Polkawallet](https://polkawallet.io/)——Polkadot在iOs和Android上使用的移动钱包。目前仍在开发之中，但测试版已经可以下载。关注GitHub上的开发进度。
```
Comment (ansonla3) [inline docs/lang/chinese/build/tools/index.md:15]:
```suggestion
- [Polkawallet](https://polkawallet.io/) - Polkadot在iOs和Android上使用的移动钱包。目前仍在开发之中，但测试版已经可以下载。关注GitHub上的开发进度。
```

File: docs/lang/chinese/build/tools/index.md [hunk 40]
```diff
@@ -0,0 +1,82 @@
...
+- [SpeckleOS](https://www.speckleos.io/)——浏览器扩展钱包，目前仍在开发之中。关注[GitHub]()上的开发进度。
```
Comment (ansonla3) [inline docs/lang/chinese/build/tools/index.md:16]:
```suggestion
- [SpeckleOS](https://www.speckleos.io/) - 浏览器扩展钱包，目前仍在开发之中。关注[GitHub](https://github.com/SpeckleOS)上的开发进度。
```

File: docs/lang/chinese/build/tools/index.md [hunk 41]
```diff
@@ -0,0 +1,82 @@
...
+- [ImageWallet](https://www.reddit.com/r/dot/comments/ah3py1/introducing_alpha_version_of_httpsimagewalletio/)——可视化跨链钱包。目前仍在开发之中。**计划支持 Polkadot和所有Substrate链**。
```
Comment (ansonla3) [inline docs/lang/chinese/build/tools/index.md]: - [ImageWallet](https://www.reddit.com/r/dot/comments/ah3py1/introducing_alpha_version_of_httpsimagewalletio/) - 可视化跨链钱包。目前仍在开发之中。**计划支持 Polkadot和所有Substrate链**。

File: docs/lang/chinese/build/tools/index.md [hunk 42]
```diff
@@ -0,0 +1,82 @@
...
+
+ 
+
+## 网络监控与报告
+
+- [Polkadot遥测服务](https://telemetry.polkadot.io/)——提供网络信息，包括当前正在运行链的节点、节点正在运行的软件版本、同步状态，以及显示节点位置的地图。
```
Comment (ansonla3) [inline docs/lang/chinese/build/tools/index.md:23]:
```suggestion
- [Polkadot遥测服务](https://telemetry.polkadot.io/) - 提供网络信息，包括当前正在运行链的节点、节点正在运行的软件版本、同步状态，以及显示节点位置的地图。
```

File: docs/lang/chinese/build/tools/index.md [hunk 43]
```diff
@@ -0,0 +1,82 @@
...
+- Polkabot——Polkadot使用Riot聊天来进行网络监控和报告，用户可以自定义创建bot插件。[博文](https://medium.com/polkadot-network/polkabot-a3dba18c20c8)，[Github仓库](https://gitlab.com/Polkabot/polkabot)
```
Comment (ansonla3) [inline docs/lang/chinese/build/tools/index.md:24]:
```suggestion
- Polkabot - Polkadot使用Riot聊天来进行网络监控和报告，用户可以自定义创建bot插件。[博文](https://medium.com/polkadot-network/polkabot-a3dba18c20c8)，[Github仓库](https://gitlab.com/Polkabot/polkabot)
```

File: docs/lang/chinese/build/tools/index.md (hunk 41 above)
Comment (ansonla3) [inline docs/lang/chinese/build/tools/index.md:17]:
```suggestion
- [ImageWallet](https://www.reddit.com/r/dot/comments/ah3py1/introducing_alpha_version_of_httpsimagewalletio/) - 可视化跨链钱包。目前仍在开发之中。**计划支持 Polkadot和所有Substrate链**。
```

**Review by ansonla3 — APPROVED** [review]

**Pull request merged**